pebblo/app/config/config_validation.py

```diff
@@ -42,6 +42,25 @@ def validate(self):
             )
 
 
+class StorageConfig(ConfigValidator):
+    def validate(self):
```

Comment:
This could be one candidate place to emit the deprecation warning for `file`

Reply:
Added.